src/modules/addon/AddonManagementDialog.cpp

```diff
@@ -154,7 +154,7 @@ AddonManagementDialog::AddonManagementDialog(QWidget * p)
 	tb = new QToolButton(hb);
 	tb->setIcon(*(g_pIconManager->getBigIcon(KVI_BIGICON_OPEN)));
 	tb->setIconSize(QSize(32,32));
-	KviTalToolTip::add(tb,__tr2qs_ctx("Install Addon Package From Disk","addon"));
+	KviTalToolTip::add(tb,__tr2qs_ctx("Install Addon Package from Disk","addon"));
```

Comment:
If you're still following https://techbase.kde.org/Projects/Usability/HIG/Capitalization I don't think tooltip is a title...